src/main/telemetry/crsf.c

```diff
@@ -267,17 +267,31 @@ void crsfFrameFlightMode(sbuf_t *dst)
     sbufWriteU8(dst, 0);
     sbufWriteU8(dst, CRSF_FRAMETYPE_FLIGHT_MODE);
 
-    // use same logic as OSD, so telemetry displays same flight text as OSD
-    const char *flightMode = "ACRO";
-    if (airmodeIsEnabled()) {
-        flightMode = "AIR";
-    }
-    if (FLIGHT_MODE(FAILSAFE_MODE)) {
-        flightMode = "!FS";
-    } else if (FLIGHT_MODE(ANGLE_MODE)) {
-        flightMode = "STAB";
-    } else if (FLIGHT_MODE(HORIZON_MODE)) {
-        flightMode = "HOR";
+    // just do "OK" for the moment as a placeholder
+    const char *flightMode = "OK";
+    if (ARMING_FLAG(ARMED)) {
+        if (isAirmodeActive()) {
+            flightMode = "AIR";
+        } else {
+            flightMode = "ACRO";
+        }
+        if (FLIGHT_MODE(FAILSAFE_MODE)) {
+            flightMode = "!FS!";
+        } else if (FLIGHT_MODE(PASSTHRU_MODE)) {
+            flightMode = "MANU";
+        } else if (FLIGHT_MODE(GPS_RESCUE_MODE)) {
+            flightMode = "RTH";
+        } else if (FLIGHT_MODE(ANGLE_MODE)) {
+            flightMode = "ANGL";
```

Comment:
This breaks compatibility with the old implementation and will break the functionality for existing users. Please change this to "STAB".

Reply:
Ah, Betaflight still uses `STAB`?  I had thought it was changed to `ANGL`.  I'll change it to `STAB`.